test/unit/rest-converter-test.js

```diff
@@ -66,10 +80,46 @@ suite('unit - converter - Rest to Tx', function() {
       assert.deepEqual(transaction.tx_json.SendMax, {
         value: '10.1',
         currency: 'USD',
-        issuer: addresses.VALID
+        issuer: addresses.VALID2
       });
       done();
     });
   });
 
+  test('convert() -- payment with same currency for source and destination, no issuer for source amount', function(done) {
```

Comment:
Need to add a test for `payment with same currency for source and destination, source amount issuer is source account, no issuer for destination amount`